types/aqb/index.d.ts

```diff
@@ -0,0 +1,1773 @@
+     */
+    update(updateOrReplaceExpr: any): UpsertRetInsertRetUpdateRet;
+    replace: UpsertRetInsertRet["update"];
+}
+export declare interface UpsertRetInsertRetUpdateRet {
+    /**
+     *
+     * @param inCollection
+     * @return
+     */
+    into(inCollection: any): UpsertExpression;
+    in: UpsertRetInsertRetUpdateRet["into"];
+    in_: UpsertRetInsertRetUpdateRet["into"];
+}
+export declare interface InsertRet {
+    /**
+     *
+     * @param collection
+     * @return
+     */
+    into(collection: any): InsertExpression;
+    in: InsertRet["into"];
+    in_: InsertRet["into"];
+}
+export declare interface UpdateRetWithRet {
+    /**
+     *
+     * @param collection
+     * @return
+     */
+    into(collection: any): UpdateExpression;
+    in: UpdateRetWithRet["into"];
+    in_: UpdateRetWithRet["into"];
+}
+export declare interface ReplaceRetWithRet {
+    /**
+     *
+     * @param collection
+     * @return
+     */
+    into(collection: any): ReplaceExpression;
+    in: ReplaceRetWithRet["into"];
+    in_: ReplaceRetWithRet["into"];
+}
+export declare interface RemoveRet {
+    /**
+     *
+     * @param collection
+     * @return
+     */
+    into(collection: any): RemoveExpression;
+    in: RemoveRet["into"];
+    in_: RemoveRet["into"];
+}
+export declare interface UpsertRet {
+    /**
+     *
+     * @param insertExpr
+     * @return
+     */
+    insert(insertExpr: any): UpsertRetInsertRet;
+}
+export declare interface UpsertRetInsertRet {
+    /**
+     *
+     * @param updateOrReplaceExpr
+     * @return
+     */
+    update(updateOrReplaceExpr: any): UpsertRetInsertRetUpdateRet;
+    replace: UpsertRetInsertRet["update"];
+}
+export declare interface UpsertRetInsertRetUpdateRet {
+    /**
+     *
+     * @param inCollection
+     * @return
+     */
+    into(inCollection: any): UpsertExpression;
+    in: UpsertRetInsertRetUpdateRet["into"];
+    in_: UpsertRetInsertRetUpdateRet["into"];
+}
+export declare interface InsertRet {
+    /**
+     *
+     * @param collection
+     * @return
+     */
+    into(collection: any): InsertExpression;
+    in: InsertRet["into"];
+    in_: InsertRet["into"];
+}
+export declare interface UpdateRetWithRet {
+    /**
+     *
+     * @param collection
+     * @return
+     */
+    into(collection: any): UpdateExpression;
+    /**
+     *
+     * @param collection
+     * @return
+     */
+    with(collection: any): UpdateRetWithRet;
+    in: UpdateRetWithRet["into"];
+    in_: UpdateRetWithRet["into"];
+}
+export declare interface ReplaceRetWithRet {
+    /**
+     *
+     * @param collection
+     * @return
+     */
+    into(collection: any): ReplaceExpression;
+    /**
+     *
+     * @param collection
+     * @return
+     */
+    with(collection: any): ReplaceRetWithRet;
+    in: ReplaceRetWithRet["into"];
+    in_: ReplaceRetWithRet["into"];
+}
+
+/**
+ *
+ * @param self
+ * @param args
+ * @return
+ */
+export declare function toArray(self: Expression, args: 1) : 1;
```

Comment:
`args` of type `1` with a return type of `1` seems suspect. Is this correct?

Reply:
That is indeed correct. Pushing the update to this.